client/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "craco start",
-    "build": "craco build",
+    "build": "GENERATE_SOURCEMAP=false craco build",
```

Comment:
let's do `GENERATE_SOURCEMAP=false yarn build` (instead of the command including the env) in our build, I think if we want to build it locally, say to test the test build, we want sourcemaps.